backend/app/controllers/game_configurations_controller.rb

```diff
@@ -17,6 +17,10 @@ def show
   def create
     @game_configuration = GameConfiguration.new(game_configuration_params)
 
+    if !(game_configuration_params[:input_set] =~ /\A([A-Za-z0-9]+ ){2,}[A-Za-z0-9]+\z/) then 
```

Comment:
See comment below about not allowing multiple of the same input items in the string.